ci-operator/step-registry/openshift/release/analysis/test-case-analysis/openshift-release-analysis-test-case-analysis-commands.sh

```diff
@@ -122,13 +127,14 @@ job-run-aggregator analyze-test-case \
 	--platform=vsphere \
 	--network=ovn \
 	--infrastructure=upi \
-	--minimum-successful-count=${MINIMUM_SUCCESSFUL_COUNT} \
```

Comment:
Is this a single job? If so, you might want to enable retry for it to make this blocking.